panda.py

```diff
@@ -0,0 +1,224 @@
+from pxr import Usd, Vt
+import json
+
+data = {}
+stage = Usd.Stage.Open('franka.usda')
+panda = stage.GetPrimAtPath('/panda')
+
+link0 = stage.GetPrimAtPath('/panda/panda_link0')
+joint1 = stage.GetPrimAtPath('/panda/panda_link0/panda_joint1')
+
+link1 = stage.GetPrimAtPath('/panda/panda_link1')
+joint2 = stage.GetPrimAtPath('/panda/panda_link1/panda_joint2')
+
+link2 = stage.GetPrimAtPath('/panda/panda_link2')
+joint3 = stage.GetPrimAtPath('/panda/panda_link2/panda_joint3')
+
+link3 = stage.GetPrimAtPath('/panda/panda_link3')
+joint4 = stage.GetPrimAtPath('/panda/panda_link3/panda_joint4')
+
+link4 = stage.GetPrimAtPath('/panda/panda_link4')
+joint5 = stage.GetPrimAtPath('/panda/panda_link4/panda_joint5')
+
+link5 = stage.GetPrimAtPath('/panda/panda_link5')
+joint6 = stage.GetPrimAtPath('/panda/panda_link5/panda_joint6')
+
+link6 = stage.GetPrimAtPath('/panda/panda_link6')
+joint7 = stage.GetPrimAtPath('/panda/panda_link6/panda_joint7')
+
+link7 = stage.GetPrimAtPath('/panda/panda_link7')
+joint8 = stage.GetPrimAtPath('/panda/panda_link7/panda_joint8')
+
+link8 = stage.GetPrimAtPath('/panda/panda_link8')
+hand_joint = stage.GetPrimAtPath('/panda/panda_link8/panda_hand_joint')
+
+hand = stage.GetPrimAtPath('/panda/panda_hand')
+finger_joint1 = stage.GetPrimAtPath('/panda/panda_hand/panda_finger_joint1')
+finger_joint2 = stage.GetPrimAtPath('/panda/panda_hand/panda_finger_joint2')
+
+print(joint1.GetPropertyNames())
+
+#Joint Limits
+joint1_uL = joint1.GetAttribute('physics:upperLimit').Get()
```

Comment:
Definitely should do a similar thing here as I described before with the for loop.